### ISSUE TITLE
[PT] Update lesson: Distillery

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2399]
## Motivation

The portuguese translation for the lesson [Distillery](https://elixirschool.com/en/lessons/libraries/distillery/) needs to be updated.
We need to update the version of the lesson from `2.0.1` to `2.0.2`.

## Proposed Solution

Translate the updates of the lesson.

You can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff 17ff3c10adb1ced03f4936efd762b24fbf6a7e9f HEAD en/lessons/libraries/distillery.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by herminiotorres** [comment]
I take this update lesson and I just sent a pull-request #2407

**Comment by thiamsantos** [comment]
Closed by #2407 

Thanks @herminiotorres :tada: :metal:
